lm-agent/tests/workload_managers/slurm/test_reservations.py

```diff
@@ -1,18 +1,20 @@
+# ruff: noqa: E501
```

Comment:
I think it works best setting a per-file ignore on the pyproject.toml level than on the file header.

Reply:
Done!